### DISCUSSION TITLE
Add note to Zavala on iPad

### DISCUSSION THREAD

**Discussion by SamSandq** [vincode-io/Zavala#163] [Support] [answered]
I have been unable to find how to add a note to an outline entry on iPad without keyboard. Please document this as well!

**Comment by vincode-io** [comment, marked as answer]
There is a context menu when you long press on a row. The option is there.

**Comment by SamSandq** [comment]
Tested it, and it works. I got confused because first you get the ’bubble’
to place the cursor, then you get the keyboard showing up. Then it’s
necessary to lift the finger and long press again; the formatting (black)
menu turns up, as well as the context menu with the add note option.

Thanks for the tip!

Cheers,
Sam

On Fri 27. May 2022 at 20.47, Maurice Parker ***@***.***>
wrote:

> There is a context menu when you long press on a row. The option is there.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vincode-io/Zavala/discussions/163#discussioncomment-2835892>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMAVI5S64EZW2NKQ37KROWLVMEDEVANCNFSM5XDPBLVQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
——————————
Sam Sandqvist
Olofsborg 4
10710 Snappertuna
Tfn. +358 50 5003597
